stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -344,6 +344,9 @@ spec:
           {{ $key }}: {{ $value }}
         {{- end }}
       spec:
+      {{- if .Values.persistentVolume.volumeName }}
+        volumeName: "{{ .Values.persistentVolume.volumeName }}"
```

Comment:
How would you have more than one replica with `volumeName` hardcoded to be the same among all?

Reply:
You are correct. That was my mistake. It could be corrected by adding a numeric suffix to the volume name and making it clear in `values.yaml` comments what is going to happen to the `volumeName` as it's used to created a sequence of volume specs.